### DISCUSSION TITLE
issues? the alarm only shows a notification and doesn't ring nor vibrate when screen is on

### DISCUSSION THREAD

**Discussion by aless3** [FossifyOrg/General-Discussion#638] [Q&A] [answered]
the alarm only shows a notification and doesn't ring nor vibrate when screen is on. 
I don't know much about android, and I don't know kotlin, but maybe it is an intended feature? 
if it is not an intended feature and it is a bug: there is a notification I can dismiss but it doesn't make any sound nor makes the phone vibrate 

I'm on a oneplus 8t, custom rom (nameless OS 13) with Android 13, sdk version 33
other clocks work, like Google clock and a few random clocks on play store, so it's not a rom issue 

this may hint it's an intended feature? not sure what the handler does thought... 
https://github.com/FossifyOrg/Clock/blob/master/app%2Fsrc%2Fmain%2Fkotlin%2Forg%2Ffossify%2Fclock%2Freceivers%2FAlarmReceiver.kt#L31-L35

**Comment by Aga-C** [comment, marked as answer]
It should have sound and vibration. If it doesn't work on your device, please raise a bug for it.
